script/main.py

```diff
@@ -129,22 +133,28 @@ def create_users_table(conn):
 
 
 def insert_users_to_table(conn, users):
-  sql = '''INSERT INTO users VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)'''
+  sql = '''INSERT INTO users VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)'''
 
   cur = conn.cursor()
   for columns in users:
     cur.execute(sql, columns)  
   conn.commit()
 
 
-def init_db(conn):  
+def init_db(conn, people_json):  
   '''Initializing the database'''    
 
   # create table
   if conn is not None:
     create_users_table(conn)
   else:
-    print("Error! cannot create the database connection.")
+    print("Error! cannot create the database connection.")  
```

Comment:
It is not enough to print error.
But I think in this case it is ok to assume that conn is not None. 
so this check would not be necessary